albatross/core/model.h

```diff
@@ -63,35 +65,47 @@ using FoldIndexer = std::map<FoldName, FoldIndices>;
  * A combination of training and testing datasets, typically used in cross
  * validation.
  */
-template <class Feature>
+template <typename FeatureType>
 struct RegressionFold {
-  RegressionDataset<Feature> train;
-  RegressionDataset<Feature> test;
+  RegressionDataset<FeatureType> train_dataset;
+  RegressionDataset<FeatureType> test_dataset;
   FoldName name;
   FoldIndices test_indices;
 
-  RegressionFold(const RegressionDataset<Feature> &train_,
-                 const RegressionDataset<Feature> &test_,
+  RegressionFold(const RegressionDataset<FeatureType> &train_dataset_,
+                 const RegressionDataset<FeatureType> &test_dataset_,
                  const FoldName &name_, const FoldIndices &test_indices_)
-      : train(train_), test(test_), name(name_), test_indices(test_indices_){};
+      : train_dataset(train_dataset_), test_dataset(test_dataset_), name(name_), test_indices(test_indices_){};
 };
 
 /*
  * A model that uses a single Feature to estimate the value of a double typed
  * target.
  */
-template <class Feature>
+template <typename FeatureType>
 class RegressionModel : public ParameterHandlingMixin {
  public:
-  RegressionModel() : ParameterHandlingMixin(), has_been_fit_(false){};
+  typedef FeatureType Feature;
```

Comment:
doesn't look like you are using this type alias anywhere.

Reply:
That's true for `albatross`, but it is being used in our private repo.